### PR TITLE
Text utilities & lists

### PR DESCRIPTION
**Pull request by laxmikanthkonthum**
Note: I think my font doesn't support all weights and sizes so i will look for a new one. Any suggestions?
Also, i will include my individual style sheets into a common one and then include this common one in all of the components at the end.
Normal lists:
image
Stacked lists: 
image
Headings: 
image
Small Text:
image
font-weights:
image
Gray Text:
image
Center-Text:
image

### CODE REVIEW

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

File: text-utilities and Lists/typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <h1 class="text-size-xl"> Headine 1 </h1>
+    <h2 class="text-size-lg"> Headine 2 </h2>
+    <h3 class="text-size-md"> Headine 3 </h3>
+    <h4 class="text-size-sm"> Headine 4 </h4>
+    <h5 class="text-size-xs"> Headine 5 </h5>
+    <p class="text-size-xl"> Text 1 </p>
+    <p class="text-size-lg"> Text 2 </p>
+    <p class="text-size-md"> Text 3 </p>
+    <p class="text-size-sm"> Text 4 </p>
+    <p class="text-size-xs"> Text 5 </p>
```
Comment (SanketDhabarde) [inline text-utilities and Lists/typography/typography.html]: I think no need to write `text-size-xl`. Even if you write `text-xl` it is self-explanatory that we are talking about size Just a small suggestion.

**Review by SanketDhabarde — CHANGES_REQUESTED** [review]
Added a small suggestion. Everything else looks good to me.

File: text-utilities and Lists/typography/typography.html (hunk 1 above)
Comment (laxmikanthkonthum) [reply text-utilities and Lists/typography/typography.html]: Right! makes sense. Will update the code - Thanks Sanket

**Pull request merged**